core/liblwm2m.c

```diff
@@ -328,3 +374,32 @@ int lwm2m_step(lwm2m_context_t * contextP,
 
     return 0;
 }
+
+#ifdef LWM2M_CLIENT_MODE
+int lwm2m_start(lwm2m_context_t * contextP)
```

Comment:
I think, don't change too much.
Otherwise a "feature" pull requests will lasts for ever :-)